cmd/osv-scanner/main_test.go

```diff
@@ -1314,6 +1318,56 @@ Filtered 2 vulnerabilities from output
 						]
 					}
 				],
+				"results_by_purl": {
+					"pkg:npm/babel@6.23.0": {
+						"Name": "babel",
+						"Version": "6.23.0",
+						"Ecosystem": "npm",
+						"Locations": [
+							{
+								"block": {
+									"file_name": "<rootdir>/fixtures/locks-licenses/package-lock.json",
+									"line_start": 13,
+									"line_end": 23,
+									"column_start": 0,
+									"column_end": 0
```

Comment:
These are a bit weird to see (columns should not be set to 0 as we detected lines) but I will investigate that in the next PR.

_Note : This comment applies to all the columns start/end set to 0 of this file_